iap/iap_test.py

```diff
@@ -57,7 +57,9 @@ def test_main(capsys):
 
     jwt_validation_result = validate_jwt.validate_iap_jwt(
         iap_jwt, expected_audience)
+    out, _ = capsys.readouterr()
 
+    assert 'JWT valdiation error' not in out
```

Comment:
I don't think that stdout contains the error message.

https://github.com/GoogleCloudPlatform/python-docs-samples/blob/dfe31fb7d5d36daf406f2c5a071e37df8dc7865b/iap/validate_jwt.py#L46

It's returning a tuple and `jwt_validation_result[2]` has the error message.

Reply:
Those two lines are wrong. It looks like they were put in by the merge with master. I'll take them back out.